src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #F3F3F3;
+  background-color: #FFF;
+  border-radius: 5px;
```

Comment:
add padding: 32px 16px 16px to your card to avoid using right/left margin for card children